functions/src/modules/SE.functions.ts

```diff
@@ -91,71 +91,68 @@ export const processNewAllotment = functions.database
 //
 /////////////////////////////////////////////////
 
-export const uploadProcessing = functions.storage.object()
+export const uploadProcessing = functions.storage.bucket(functions.config().sound_editing.restoration.uploads.bucket_name).object()
 .onFinalize(async object => {    
 
     const filePath = object.name;
-
-
-    if (!filePath.startsWith('sound-editing/restored/uploads'))
-        return -1;
+    const seUploadsBucket = admin.storage().bucket(functions.config().sound_editing.restoration.uploads.bucket_name);    
```

Comment:
Do we really need to define the bucket again? Can't we get it from the `object`?